content/docs/0.8.1/snapshots-and-backups/backup-and-restore/backupstores-and-backuptargets.md

```diff
@@ -108,6 +108,13 @@ Please follow [the Kubernetes document](https://kubernetes.io/docs/concepts/conf
 
 Notice the secret must be created in the `longhorn-system` namespace for Longhorn to access.
 
+### Using a self signed SSL certificate for S3 communication
+If you want to use a self signed ssl certificate you can expose an AWS_CERT environment variable that contains your public certificate to the longhorn pods.
+we recommend you do so by adding the AWS_CERT key/val to the `minio-secret` or `aws-secret` and making it available to the `longhorn-system` namespace.
+For an example of using a self signed ssl certificate, have a look at our `minio-backupstore.yaml` linked above.
+It's important to note that the Certificate needs to be in PEM format and must either be a certificate authority,
+or the certificate chain including the certificate authority must be included by concatenating all relevant certificates.
```

Comment:
Grammar.

Reply:
@yasker how should the grammar be fixed? should it be something like this?

> You will need to upload your own public- or private-CA-signed certificate and the associated key as PEM-encoded files. If you are using a certificate chain, all relevant certificates must be included by concatenating them.

Reply:
@catherineluse Yes, this looks good. @joshimoo can you confirm?

Reply:
@catherineluse @yasker I like the shorter text, but it's inaccurate since the user does not upload the certificate, instead it needs to be exposed as a AWS_CERT environment variable. Also it's important to understand that the public in this part refers to the certificate and has a special (loaded) meaning in the ssl/tls/https context, the same applies to the CA (certificate authority) statements above.

The reason I wrote such a long text is that the details are complicated, but if we don't need to be 100% accurate and specific, we certainly can go with some simplifications. :)

Here is some context information that might help: https://superuser.com/questions/620121/what-is-the-difference-between-a-certificate-and-a-key-with-respect-to-ssl

Reply:
Is the following accurate?
> It's important to note that the certificate needs to be in PEM format and must either be a certificate authority, or a certificate chain concatenated with all relevant certificates, which includes all the certificate authorities.

Reply:
I think we can go with your text, technically I think the or is referring to "the certificate" which would be inaccurate. since in the or case we are concatenating multiple certificates, etc together.

Pem itself is a container format that may include just the public certificate (such as with Apache installs, and CA certificate files /etc/ssl/certs), or may include an entire certificate chain including public key, private key, and root certificates.

Reply:
After a lengthy discussion with @joshimoo , we've agreed on using the following sentence: 
> It's important to note that the certificate needs to be in PEM format, and must be it's own CA. Or one must include a certificate chain that contains the CA certificate. 
To include multiple certificates, one can just concatenate the different certificates (PEM files)